invenio_rdm_records/services/schemas/stats.py

```diff
@@ -2,17 +2,28 @@
 #
 # Copyright (C) 2020 CERN.
 # Copyright (C) 2020 Northwestern University.
+# Copyright (C) 2023 TU Wien.
 #
 # Invenio-RDM-Records is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
 
-"""RDM record schemas."""
+"""Schema for record statistics."""
 
 from marshmallow import Schema, fields
 
 
-class StatsSchema(Schema):
-    """Files metadata schema."""
+class PartialStatsSchema(Schema):
```

Comment:
This would be the schema for 1 version right? What about `RecordStatsSchema`? No need for name change, otherwise, just add in the docstring to "what" is it partial.

Reply:
yes, it's the shared schema between "this version" and "all versions" because both look the same...
i was also toying with the idea of `RecordStatsSchema` but figured it might be confusing which one to pick (`StatsSchema` or `RecordStatsSchema`?) if somebody wanted to import it :sweat_smile:

Reply:
updated the docstring